functable.c

```diff
@@ -2,11 +2,12 @@
  * Copyright (C) 2017 Hans Kristian Rosbach
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
+#ifndef DISABLE_RUNTIME_CPU_DETECTION
 
 #include "zbuild.h"
-#include "zendian.h"
```

Comment:
This dependency can be removed from win32/ makefiles. And the one below should be added.

Reply:
This dependency is not direct now, but indirect, through ``cpu_functions.h``.

As I see the dependencies in ``win32/Makefile.*`` were broken before this PR.
For example, ``win32/Makefile.arm`` has a dependency on x86 file ``$(SRCDIR)/arch/x86/x86_features.h``.

If we need actual nmake support (without cmake), I suggest fix the nmake-dependencies with separate PRs.

Reply:
I can try to fix the nmake dependencies when all the "more important" changes have been merged... It would help a lot if people would create issues when there is something wrong in the nmake Makefiles as I can't track all the changes if pull requests are merged in batches. I try to check most of the new pull requests at least once a week (unless I'm tagged in the pull request), but my time is limited due to other projects that are more important.